docs/machine-learning/tutorials/tensorflow-transfer-learning.md

```diff
@@ -0,0 +1,337 @@
+
+
+
+[!code-csharp[Classifications](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#Classifications)]
+
+## Results
+
+Your results should be similar to the following output. As the pipeline processes, it displays messages. You may see warnings, or processing messages. These have been removed from the following results for clarity.
+
+```console
+Images folder: C:\Tutorial\TransferLearningTF\TransferLearningTF\bin\Debug\netcoreapp2.2\assets\inputs-train\data
+Training file: C:\Tutorial\TransferLearningTF\TransferLearningTF\bin\Debug\netcoreapp2.2\assets\inputs-train\data\tags.tsv
+Default parameters: image size=(224,224), image mean: 117
+2019-03-18 20:18:15.235693: I tensorflow/core/platform/cpu_feature_guard.cc:141] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
+=============== Training classification model ===============
+ImagePath: pizza.jpg predicted as: pizza with score: 0.9844866
+ImagePath: pizza2.jpg predicted as: pizza with score: 0.9481499
+ImagePath: teddy2.jpg predicted as: teddy with score: 0.9762651
+ImagePath: teddy3.jpg predicted as: teddy with score: 0.9821286
+ImagePath: teddy4.jpg predicted as: teddy with score: 0.9862462
+ImagePath: toaster.jpg predicted as: toaster with score: 0.977837
+ImagePath: toaster2.png predicted as: toaster with score: 0.9796767
+=============== Classification metrics ===============
+LogLoss is: 0.0239608645879546
+PerClassLogLoss is: 0.0344388607695228 , 0.0186344570300533 , 0.0214724797432381
+=============== Save model to local file ===============
+Model saved: C:\Tutorial\TransferLearningTF\TransferLearningTF\bin\Debug\netcoreapp2.2\assets\outputs\imageClassifier.zip
+=============== Loading model ===============
+Model loaded: C:\Tutorial\TransferLearningTF\TransferLearningTF\bin\Debug\netcoreapp2.2\assets\outputs\imageClassifier.zip
+=============== Making classifications ===============
+ImagePath: broccoli.png predicted as: teddy with score: 0.5549887
+ImagePath: pizza3.jpg predicted as: pizza with score: 0.9718443
+ImagePath: teddy6.jpg predicted as: teddy with score: 0.9809543
+ImagePath: toaster3.jpg predicted as: toaster with score: 0.9649727
+Press any key to continue . . .
+```
+
+Congratulations! You've now successfully built a machine learning model for image classification by reusing a pre-trained TensorFlow model.
+
+You can find the source code for this tutorial at the [dotnet/samples](https://github.com/dotnet/samples/tree/master/machine-learning/tutorials/TransferLearningTF) repository.
+
+In this tutorial, you learned how to:
+> [!div class="checklist"]
+> * Understand the problem
+> * Build and Train the model
+> * Predict with the trained model
+> * Deploy and Predict with a loaded model
+
+Check out the Machine Learning samples GitHub repository to explore an expanded image classification sample.
```

Comment:
Next steps here should be trying out different styles of Transfer Learning or other model architectures.